src/APL/redis-apl.test.ts

```diff
@@ -0,0 +1,56 @@
+import { afterEach, describe, expect, it } from "vitest";
+
+import { AuthData } from "./apl";
+import { RedisAPL } from "./redis-apl";
+
+// Obviously, for this test to pass you need to have a docker container running redis :)
```

Comment:
In units tests we will not have Redis available so you need to mock it, otherwise your PR will kill CICD until we rework infrastructure